pipelines/amplicon-snp-calling-parasite/input_files/RCN15267.json

```diff
@@ -0,0 +1,19 @@
+{
+  "AmpliconSNPCallingParasite.input_bam": "gs://broad-dsde-methods-malariagen/parasite/test-data/RCN15267.PfD37.cram",
+  "AmpliconSNPCallingParasite.input_bam_index": "gs://broad-dsde-methods-malariagen/parasite/test-data/RCN15267.PfD37.cram.crai",
+  "AmpliconSNPCallingParasite.output_basename": "RCN15267",
+  "AmpliconSNPCallingParasite.reference": {
+    "ref_fasta": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta",
+    "ref_dict": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.dict",
+    "ref_ann": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.ann",
+    "ref_sa": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.sa",
+    "ref_amb": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.amb",
+    "ref_bwt": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.bwt",
+    "ref_fasta_index": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.fai",
+    "ref_pac": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.fasta.pac"
+  },
+  "AmpliconSNPCallingParasite.reference_ploidy": "gs://broad-dsde-methods-malariagen/parasite/references/PfD37/Pfalciparum_contigs.ploidy",
```

Comment:
Hmm, I just noticed and removed this extra line and force pushed the branch. However, while it doesn't appear when you view the branch outside of the PR, it still appears here in the PR. Have I done something wrong, or is GitHub on the fritz?

Reply:
Might indeed be on the fritz: https://www.githubstatus.com/incidents/phnch1rww464?utm_ts=1594827674

Reply:
OK, looks to be resolved now.